pkg/certificate/store/store_darwin.go

```diff
@@ -0,0 +1,223 @@
+//go:build darwin
+
+package store
+
+import (
+	"bytes"
+	"crypto/x509"
+	"encoding/asn1"
+	"encoding/pem"
+	"fmt"
+	"io/fs"
+	"os"
+	"os/exec"
+
+	"github.com/massalabs/station/pkg/runner"
+	"howett.net/plist"
+)
+
+var _ runner.Runner = &SecurityRunner{}
+
+const (
+	permissionUrwGrOr = 0o644
+	permissionUrw     = 0o600
+
+	tmpTrustSettingsFile = "trust-settings.plist"
+	tmpTrustedCertFile   = "trusted-cert"
```

Comment:
not sure that the temporary notion brings a significant piece of information here.